module/VuFind/src/VuFind/RecordTab/HoldingsILS.php

```diff
@@ -93,6 +93,25 @@ public function getDescription()
         return 'Holdings';
     }
 
+    /**
+     * Is this tab initially visible?
+     *
+     * @return bool
+     */
+    public function isVisible()
+    {
+        // if the record source is EDS, assume hidden holdings tab
+        // unless it is a catalog record and the publication type
+        // is not excluded from RTAC
+        if ($this->driver->getSourceIdentifier() === 'EDS') {
+            return
+                $this->driver->hasCatalog() &&
+                $this->driver->pubTypeRtacEnabled();
```

Comment:
I wonder if we could eliminate the EDS-specific check here by coming up with a more generic method name like `supportsHoldingsTab`. Then we could do something like `return $this->driver->tryMethod('supportsHoldingsTab', true);` so that the tab displays by default unless the record driver says it's not compatible, and you could move this logic into the EDS driver itself for more flexibility/encapsulation.

Reply:
I followed your guidance and moved this to the driver and agree, this makes it cleaner and less dependent on it being an EDS "feature", which I like